src/configuration/vimrc.ts

```diff
@@ -18,6 +18,60 @@ export class VimrcImpl {
     return this._vimrcPath;
   }
 
+  private static readonly SOURCE_REG_REX = /^(source)\s+(.+)/i;
+
+  private static buildSource(line: string) {
+    if (line.trimLeft().startsWith('"')) {
```

Comment:
Instead of having this same check in every `build*` function, let's just put it at the start of the loop over lines in `loadConfig`